src/styles/_components/_header.scss

```diff
@@ -254,42 +254,12 @@ $header-height: 60px;
         position: relative;
         display: none;
         z-index: 3;
-        padding-right: 20px;
         @include font("IntervalNextReg");
         font-size: em(15px);
         text-transform: uppercase;
 
-        &:before,
-        &:after {
-            content: " ";
-            position: absolute;
-            display: block;
-            top: 11px;
-            right: 0;
-
-            width: 8px;
-            height: 1px;
-
-            background-color: color('feedback', 'text');
-        }
-
-        &:before {
-            transform: rotate(-45deg);
-        }
-
-        &:after {
-            right: 5px;
-            transform: rotate(45deg);
-        }
-
-        &--active {
-            &:before {
-                transform: rotate(45deg);
-            }
-
-            &:after {
-                transform: rotate(-45deg);
-            }
+        svg {
+            fill: #0083e4;
```

Comment:
Are these colors going to be used elsewhere?

Reply:
@AndreJoaquim,  as addressed in our last sync new colours will not be added nor the old one's changes until we settle on a new approach for with the design team. But answering your question, no, this is the first time we are using this colour YAG 💪